cypress/e2e/shared/scriptQueryBuilder.sql.test.ts

```diff
@@ -42,35 +49,41 @@ describe('Script Builder', () => {
   }
 
   before(() => {
-    const writeData: string[] = []
-    for (let i = 0; i < 30; i++) {
-      writeData.push(`ndbc,air_temp_degc=70_degrees station_id_${i}=${i}`)
-      writeData.push(`ndbc2,air_temp_degc=70_degrees station_id_${i}=${i}`)
+    const generateWriteData = (value: number) => {
+      return [
+        `${measurement},${tagKey}=${tagValue} ${fieldName}=${value}`,
+        `${measurement},${tagKey}=${tagValue} ${fieldName2}=${value}`,
+        `${measurement},${tagKey}=${tagValue2} ${fieldName}=${value}`,
+        `${measurement},${tagKey}=${tagValue2} ${fieldName2}=${value}`,
+      ]
```

Comment:
This is making our DB data look more like real data. (Rather than the previous version used for test fixtures, where we only cared about filling out a raw table.)